opal-shell/src/main/java/org/obiba/opal/shell/commands/AnalyseCommand.java

```diff
@@ -64,8 +64,6 @@
 
   @Override
   public int execute() {
-    // TODO validate subject permissions
-
```

Comment:
Values level access to analyzed tables needs to be verified.

Reply:
I don't think I need that, I did tests on DS-basis permission and per-table-basis permission and the code runs correctly. If I run a batch of two analysis on two tables, only the one that has the proper permission gets analysed. If I give the right permission at DS level, all tables get analysed. May be I am missing another case?